Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -795,6 +795,12 @@ def audit_specs
         return if stable_url_minor_version.even?
 
         problem "#{stable.version} is a development release"
+      when %r{^https://github.com/}
+        _, owner = URI.parse(stable.url).path.split("/")
+        tag = stable.specs[:tag] || stable.version.to_s
```

Comment:
That won't work for probably most formulae where `version` does not match the actual tag in the project repository.

Reply:
I wouldn't say that's most formulae but it will be some, sure. Any suggestions @dawidd6?

Reply:
Would need to query the API for available tags and match the formula version, I think.

Reply:
That ends up being equivalent in this case: it'll 404 if the tag doesn't exist (which we can treat as "not a prerelease")

Reply:
Would it make sense to instead check what is given to `url` and get the tag from that?

Reply:
@Bo98 You mean match it with a regex? I like that idea.

Reply:
Yeah, whatever is necessary to turn `https://github.com/org/repo/archive/v1.0.0.tar.gz` or `https://github.com/org/repo/releases/download/v1.0.0/project-1.0.0-src.tar.gz` into `v1.0.0`.

I don't think it makes any sense to use `stable.version` and try to reverse what the parsing does to remove the `v`.

Of course, in the case of git cloning, we can use `stable.specs[:tag]`.

Reply:
Yeh, that seems like a good idea 👍